rest_api_flint.example/app.py

```diff
@@ -7,10 +7,14 @@
 from datetime import datetime
 from flask_swagger import swagger
 from flask_swagger_ui import get_swaggerui_blueprint
+from flask_cors import CORS
 
 app = Flask(__name__)
+app = Flask(__name__)
+CORS(app)
```

Comment:
Can we add the origins like this for local host and other deployed links?

Example:
`cors = CORS(app, origins=['http://localhost:5000',r'^https://.+example.com$'])`

Reply:
I am not sure how would the two containers (API and the UI) will talk to each other, and so I can not fix the domains for now. Can we do this once the IP for the UI container is fixed? 😅
`CORS(app)` enables CORS on all domains I suppose.

Reply:
Start with localhost for now and we'll add more origins as needed. I think this approach forces people to 'opt-in' to CORS when setting up their domain, which is better practice than enabling it by default and having people 'opt-out'.

Reply:
Done!